packages/headless/src/utils/facet-utils.ts

```diff
@@ -0,0 +1,11 @@
+export function sortFacets<T extends {facetId: string}>(
```

Comment:
This is called with `T` as `FacetManagerPayload` and `T` as `AnyFacetRequest`.

Reply:
Is the generic really required here ? 

I think it could just be : 


```
export function sortFacets(facets: {facetId: string}[], facetIds: string[]) {
 [...]
}
```

Reply:
Not required, I'll change it.

Reply:
It's required for the return type.